var/spack/repos/builtin/packages/wcstools/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Wcstools(MakefilePackage):
+    """
+    Astronomers often need to relate positions on an image of the sky to positions
+    on the real sky to identify catalogued objects in images, tell other people where
+    to look to find an identified object, or to compute motions of planets, satellites,
+    asteroids, or comets.
+    WCSTools is a package of programs and a library of utility subroutines for setting and
+    using the world coordinate systems (WCS) in the headers of the most common astronomical
+    image formats, FITS and IRAF .imh, to relate image pixels to sky coordinates.
+    This software is all written in very portable C, so it should compile and
+    run on any computer with a C compiler.
+    """
+
+    homepage = "http://tdc-www.harvard.edu/wcstools/"
+    url = "http://tdc-www.harvard.edu/software/wcstools/wcstools-3.9.7.tar.gz"
+    maintainers("pelahi")
+    version("3.9.7", sha256="525f6970eb818f822db75c1526b3122b1af078affa572dce303de37df5c7b088")
```

Comment:
Conventional line spacing.
```suggestion

    maintainers("pelahi")

    version("3.9.7", sha256="525f6970eb818f822db75c1526b3122b1af078affa572dce303de37df5c7b088")
```